### DISCUSSION TITLE
bug

### DISCUSSION THREAD

**Discussion by AMDGOOD** [husandro/WoWTools#5] [General]
UP你好，聊天框里把角色名改成“我”这个功能和爱不易捐献图标冲突，会显示成：
08:38:11 [队长] [|TInterface\AddOns\!!!163UI!!!\Textures\UI2*]: 1

**Comment by husandro** [comment]
明天修

**Reply by AMDGOOD** [reply]
感谢，另外更新方式是不是只有在这里下载获取？

**Reply by AMDGOOD** [reply]
更新了你的插件后上线试了试，还是不行呢。

**Reply by husandro** [reply]
你好, 你使用的是整合插件,  我下载了爱不易, 找不到是哪插件(爱不易中), 试了一下, 不知问题出在哪里, 
你可以把我的超链接功能关了试试,新版本是最右边按钮,

**Reply by AMDGOOD** [reply]
额，我是爱不易有捐献的话聊天框发言名字前面会带个图标，例如[X奶骑]，X=图标。没在捐献名单里没这个功能，不过也没有开关。

**Reply by AMDGOOD** [reply]
试了试，禁用wowtool后进游戏，然后再加载wowtool，两个功能是可以共存的，也就是显示[X我]，但RL一遍后又不行了。要是麻烦的话就算了大概也没多少人捐钱有这个图标...另外更新只能在这下载覆盖吗。
